ReactiveCocoaTests/Swift/PropertySpec.swift

```diff
@@ -106,6 +106,36 @@ class PropertySpec: QuickSpec {
 				property.value = 1
 				expect(value).to(equal(1))
 			}
+			
+			describe("valueTransformer") {
+				context("with the default value transformer") {
+					it("should not transform the value set") {
+						let property = MutableProperty(0)
+						var sentValue: Int?
+						
+						property.producer.startWithNext { value in
+							sentValue = value
+						}
+						
+						expect(property.value).to(equal(0))
```

Comment:
This is only testing the initial value. Maybe change the property a few times to verify that this remains true?